index.html

```diff
@@ -9,203 +8,189 @@
-                the like).</p>
-        </div>
-        <button type="button" class="colbtn">Check out my CV <img src="./Images/ic_download.png" alt=""></button>
-
-        <div class="lang">
-            <div class="sectitle">
-                <ul>
-                    <li>
-                        <h2>Languages</h2>
-                    </li>
-                </ul>
+    <section class="about" id="about">
+        <div class="about_wrapper">
+            <div class="headtitle">
+            <h1 class="mttitle" >About</h1>
+            </div>
+            <div class="mttxt">
+            <p class="mtext" >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
             </div>
-            <div class="litxt">
+            <button type="button" class="colbtn">Get my CV <img src="./Images/ic_download.png" alt=""></button>
+        </div>
+        <div class="lang_con">
+            <div class="lang">
+                <div class="sectitle">
+                    <h2>Languages</h2>
+                </div>
+                <div class="litxt">
                 <ul>
                     <li>JavaScript (ES5/ES6)</li>
                     <li>Ruby</li>
                     <li>HTML</li>
                     <li>CSS</li>
                 </ul>
+                </div>
             </div>
-        </div>
-        <div class="frames">
-            <div class="sectitle">
-                <ul>
-                    <li>
-                        <h2>Frameworks</h2>
-                    </li>
-                </ul>
-            </div>
-            <div class="litxt">
+            <div class="frames">
+                <div class="sectitle">
+                    <h2>Frameworks</h2>
+                </div>
+                <div class="litxt">
                 <ul>
                     <li>Bootstrap</li>
                     <li>Ruby on rails</li>
                     <li>RSpec</li>
                     <li>Capibara</li>
                     <li>Selenium</li>
                 </ul>
+                </div>
             </div>
-        </div>
-        <div class="skills">
-            <div class="sectitle">
-                <ul>
-                    <li>
-                        <h2>Skills</h2>
-                    </li>
-                </ul>
-            </div>
-            <div class="litxt">
+            <div class="skills">
+                <div class="sectitle">
+                    <h2>Skills</h2>
+                </div>
+                <div class="litxt">
                 <ul>
                     <li>Codekit</li>
                     <li>Codepen</li>
                     <li>GitHub</li>
                     <li>GitLab</li>
                     <li>Terminal</li>
                 </ul>
+                </div>
             </div>
         </div>
     </section>
```

Comment:
- [ ] Kindly make sure your About section design sticks to the one from the Figma template. Align everything at his expected position by respecting the **spacing**, **width**, **alignment positions**, **fonts**, **backgrounds**, **size**,colors,... Then make sure your design sticks to the Figma template. AS you can see from the images below they look different.
### Figam design
![rf_3](https://user-images.githubusercontent.com/83042742/171850199-97d55fd9-f2e4-4bce-9781-b69cb13c711a.png)
### Your design
![rs_s3](https://user-images.githubusercontent.com/83042742/171850271-50fbf842-11b7-4416-aa7b-3c78b6b420e2.png)